apps/dcos_l4lb/src/dcos_l4lb_ipvs_mgr.erl

```diff
@@ -133,30 +134,48 @@ init([]) ->
     {ok, #state{netns = #{host => Pid}, family = Family}}.
 
 handle_call({get_services, Namespace}, _From, State) ->
-    Reply = handle_get_services(Namespace, State),
+    Reply = handle_call_track_time(fun handle_get_services/2, [Namespace, State]),
     {reply, Reply, State};
 handle_call({add_service, IP, Port, Protocol, Namespace}, _From, State) ->
-    Reply = handle_add_service(IP, Port, Protocol, Namespace, State),
+    Reply = handle_call_track_time(fun handle_add_service/5, [IP, Port, Protocol, Namespace, State]),
     {reply, Reply, State};
 handle_call({remove_service, IP, Port, Protocol, Namespace}, _From, State) ->
-    Reply = handle_remove_service(IP, Port, Protocol, Namespace, State),
+    Reply = handle_call_track_time(fun handle_remove_service/5, [IP, Port, Protocol, Namespace, State]),
     {reply, Reply, State};
 handle_call({get_dests, Service, Namespace}, _From, State) ->
-    Reply = handle_get_dests(Service, Namespace, State),
+    Reply = handle_call_track_time(
+        fun handle_get_dests/3, [Service, Namespace, State]),
     {reply, Reply, State};
 handle_call({add_dest, ServiceIP, ServicePort, DestIP, DestPort, Protocol, Namespace}, _From, State) ->
-    Reply = handle_add_dest(ServiceIP, ServicePort, DestIP, DestPort, Protocol, Namespace, State),
+    Reply = handle_call_track_time(
+        fun handle_add_dest/7,
+        [ServiceIP, ServicePort, DestIP, DestPort, Protocol, Namespace, State]),
     {reply, Reply, State};
 handle_call({remove_dest, ServiceIP, ServicePort, DestIP, DestPort, Protocol, Namespace}, _From, State) ->
-    Reply = handle_remove_dest(ServiceIP, ServicePort, DestIP, DestPort, Protocol, Namespace, State),
+    Reply = handle_call_track_time(
+        fun handle_remove_dest/7,
+        [ServiceIP, ServicePort, DestIP, DestPort, Protocol, Namespace, State]),
     {reply, Reply, State};
 handle_call({add_netns, UpdateValue}, _From, State0) ->
-    {Reply, State1} = handle_add_netns(UpdateValue, State0),
+    {Reply, State1} = handle_call_track_time(fun handle_add_netns/2, [UpdateValue, State0]),
     {reply, Reply, State1};
 handle_call({remove_netns, UpdateValue}, _From, State0) ->
-    {Reply, State1} = handle_remove_netns(UpdateValue, State0),
+    {Reply, State1} = handle_call_track_time(
+        fun handle_remove_netns/2, [UpdateValue, State0]),
     {reply, Reply, State1}.
 
+handle_call_track_time(Fun, Args) ->
+    Begin = erlang:monotonic_time(),
+    Return = erlang:apply(Fun, Args),
+    try
+        prometheus_summary:observe(
+            l4lb, ipvs_updates_seconds, [],
+            erlang:monotonic_time() - Begin)
+    catch error:_Error ->
```

Comment:
In some places, such errors are simply ignored, whereas in others, they are logged. Why is there such a difference in handling them?

Reply:
The problem the `try/catch` is trying to solve is that some of the `ct` tests fail when we hit calls to store metrics. Ideally, I would like to solve that by fixing the tests, but I was not able to find a way to do so. We have used this pattern before when it meant huge test refactoring. 

If I remember correctly, we haven't been putting `loggers` on those catches whose goals is just to make a test pass. I think I might have forgotten one there, so I will make sure none of them have it

Reply:
I will resolve it, fell free to reopen if you feel like.